extensions/BLENDER_physics/schema/node.BLENDER_physics.schema.json

```diff
@@ -29,16 +29,73 @@
             "type": "boolean",
             "default": false
         },
-        "dimensions" : {
-            "description": "The bounding box dimensions of the node",
+        "bounding_box" : {
+            "description": "The local bounding box dimensions of the node",
             "type": "array",
             "items": {"type": "number"},
             "minItems": 3,
             "maxItems": 3
         },
-        "mesh": {
+        "radius" : {
+            "description": "Radius to use for calculating the physics shape based on bounding box of the node",
+            "type": "number",
+            "default": 0.0
+        },
+        "height" : {
+            "description": "Height to use for calculating the physics shape based on bounding box of the node",
+            "type": "number",
+            "default": 0.0
+        },
+        "height_axis" : {
+            "description": "The axis (0=X, 1=Y, 2=Z) to use for the height of the node",
```

Comment:
I know this is a hard field to describe, but I feel like we might be able to do a bit better than this description.